packages/ethtool/ethtool.spec

```diff
@@ -0,0 +1,30 @@
+Name:           %{_cross_os}ethtool
+Epoch:          2
+Version:        6.2
+Release:        1%{?dist}
+Summary:        Settings tool for Ethernet NICs
+License:        GPL-2.0-only AND GPL-2.0-or-later
+URL:            https://www.kernel.org/pub/software/network/ethtool/
+Source0:        https://www.kernel.org/pub/software/network/ethtool/ethtool-%{version}.tar.xz
+BuildRequires:  %{_cross_os}glibc-devel
+BuildRequires:  %{_cross_os}libmnl-devel
```

Comment:
I'd rather avoid trying to align things in columns; it's inconsistent with all the other specs, if nothing else.

It can lead to fussy whitespace-related changes in future diffs to fix up alignment. It's analogous in my mind to not using trailing commas on the final list elements (in TOML) and struct members (Rust) - it may look "nicer" but adds a maintenance chore that's otherwise not needed.

Reply:
Fixed